src/shared/components/palette/case-flag/components/update-flag/update-flag.component.html

```diff
@@ -0,0 +1,46 @@
+<div class="form-group" [formGroup]="formGroup">
+  <div class="govuk-character-count" data-module="govuk-character-count" data-maxlength="100">
+    <div class="govuk-form-group" [ngClass]="{'form-group-error': errorMessages.length > 0}">
+      <h1 class="govuk-label-wrapper"><label class="govuk-label govuk-label--l" [for]="updateFlagControlName">
+         {{updateFlagTitle}}
+        </label>
+      </h1>
+      <div id="add-comments-hint" class="govuk-hint">
```

Comment:
@tarikdogan-hmcts - please change the `id` to `"update-flag-hint"`

Reply:
done